resources/src/Table.css

```
@@ -0,0 +1,66 @@
+.text-left {
+  text-align: left;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.background-blue {
+  background: rgba(128, 187, 255, 0.25);
+}
+
+.background-green {
+  background: rgba(113, 186, 93, 0.25);
+}
+
+.background-yellow {
+  background: rgba(245, 198, 122, 0.25);
+}
+
+.background-red {
+  background: rgba(255, 128, 128, 0.25);
+}
```

Personally, I think the colors could need a bit more saturation.
```suggestion
.background-blue {
  background: rgba(128, 187, 255, 0.4);
}

.background-green {
  background: rgba(113, 186, 93, 0.4);
}

.background-yellow {
  background: rgba(245, 198, 122, 0.4);
}

.background-red {
  background: rgba(255, 128, 128, 0.4);
}
```

---

This looks indeed a little more saturated and it's easier to spot different colors👍